### PR TITLE
Feat: Add Navbar component

### LINKED ISSUE #3: Create Header

### PR DESCRIPTION
**Pull request by JosephTarrazi**
resolves #3

### CODE REVIEW

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+function NavBar() {
+  return (
+    <Box>
+      <Flex
+        justify="space-between"
+        align="center"
+        mx={16}
+        mt={3}
+        fonts="
+            Montserrat"
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: This won't have any effect - removing it is advisable

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        fontSize={16}
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: Please use the proper theme token provided to you by chakra UI in which case it's fontSize='md'

File: src/components/NavBar/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        fontWeight={600}
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: Please use the proper theme token provided to you by chakra UI in which case it's fontWeight='medium'

File: src/components/NavBar/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        bg="transparent"
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: The navbar background is transparent by default - this is redundant

File: src/components/NavBar/NavBar.jsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      >
+        <Box fontSize={24}>LOGO</Box>
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: Please use the proper theme token provided to you by chakra UI in which case it's fontSize='2xl'

File: src/components/NavBar/NavBar.jsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                  <Icon as={Globe} mr={2} /> FR
+                </Box>
+              </MenuItem>
+              <MenuItem>
+                <Box color="gray.400" _hover={{ color: 'green.400' }}>
+                  <Icon as={Globe} mr={2} /> AR
+                </Box>
+              </MenuItem>
+            </MenuList>
+          </Menu>
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: This code snippet needs quite a bit of refactoring:

- MenuList has a default "min-width: 14rem" which is far larger than the actual content itself. A quick fix would be to pass the following prop explicitly: minW="max-content"

File: src/components/NavBar/NavBar.jsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+          <Text _hover={{ color: 'green.400' }}>Home</Text>
+          <Text _hover={{ color: 'green.400' }}>Items</Text>
+          <Text _hover={{ color: 'green.400' }}>About</Text>
+          <Text _hover={{ color: 'green.400' }}>Contact us</Text>
+        </HStack>
+
+        <Flex>
+          <Menu>
+            <MenuButton
+              p={1}
```
Comment (Skia69) [inline src/components/NavBar/NavBar.jsx]: p={2} will make the button look quite better

File: src/pages/HomePage.jsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import NavBar from '../components/NavBar/NavBar';
+function HomePage() {
+  return (
+    <div>
+      <NavBar />
```
Comment (Skia69) [inline src/pages/HomePage.jsx]: NavBar should reside in App.js since it's presumably going to be present in every single page

**Review by Skia69 — CHANGES_REQUESTED** [review]
Please, take into consideration the comments I dropped below

**Comment by adlogi** [comment]
Thank you @JosephTarrazi . It looks good. You can quickly resolve Mhammad's comments and it'd be ready to merge. Please add .eslintcache to .gitignore.

**Review by adlogi — APPROVED** [review]

**Pull request merged**